README.md

```diff
@@ -7,7 +7,7 @@ There are several steps that need to be done when rebasing a package. The goal o
 
 ## General workflow
 - *rebase-helper-workspace* and *rebase-helper-results* directories are created
-- original SPEC file is copied to *rebase-helper-results* directory and its Version tag is modified
+- original SPEC file is copied to *rebase-helper-results/rebased_sources* directory and its Version tag is modified
```

Comment:
This is probably not the right place to complain about this, but here the inconsistency really stands out. `-` is used in `rebase-helper-results` as a separator while in `rebased_sources` it's `_`.